crates/typst-ide/src/jump.rs

```diff
@@ -53,10 +53,14 @@ pub fn jump_from_click(
     for (mut pos, item) in frame.items().rev() {
         match item {
             FrameItem::Group(group) => {
-                // TODO: Handle transformation.
-                if let Some(span) =
-                    jump_from_click(world, document, &group.frame, click - pos)
-                {
+                let pos = click - pos;
+                if let Some(clip) = &group.clip {
+                    if !clip.contains(pos) {
+                        continue;
+                    }
+                }
+                let pos = pos.transform_inf(group.transform.invert().unwrap());
```

Comment:
I think there's no guarantee that this is invertible since any group could be created. Better to be safe and just skip the group if the transform is not invertible instead of panicking.

Reply:
I agree. But can you describe in natural language what it would mean if a transform matrix is not invertible? Like what kind of transforms would lead to that?